crates/sui-indexer/src/handlers/objects_snapshot_processor.rs

```diff
@@ -1,10 +1,10 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
+use std::sync::{Arc, Mutex};
```

Comment:
will rename the file to `_handler` before merging, leaving it for now for PR review.